templates/pipeline/structured_error_counts_v1/view.sql

```diff
@@ -0,0 +1,45 @@
+WITH ping_counts AS (
+  SELECT
+    TIMESTAMP_TRUNC(submission_timestamp, HOUR) AS hour,
+    metadata.document_namespace,
+    metadata.document_type,
+    metadata.document_version,
+    COUNT(*) AS ping_count
+  FROM
+    `payload_bytes_decoded.structured_*`
+  WHERE
+    submission_timestamp >= TIMESTAMP_SUB(current_timestamp, INTERVAL 28 * 24 HOUR)
```

Comment:
The predicate-pushdown for `WHERE hour > ..` works, but the query processor complains if we don't explicitly limit using this kind of filter.